### DISCUSSION TITLE
Welcome to @giscus/react component!

### DISCUSSION THREAD

**Discussion by giscus** [reynoldputra/reynoldputra.com#11] [General]
# Welcome to @giscus/react component!

http://localhost:3001/guestbook

**Comment by reynoldputra** [comment]
Welcome to my guestbook! 🎉
Feel free to leave a comment by signing in with your GitHub account 😄

**Comment by wisnuyasha** [comment]
Your portfolio website looks absolutely stunning! The UI/UX design is both aesthetically pleasing and functional, with a perfect balance of color and layout. The smooth and interactive interface adds to the overall user experience, making it so satisfying. sheeeeeeeeesh

**Reply by reynoldputra** [reply]
Thank you! I hope to continue consistently improving this website 🤩

**Comment by dzakiruw** [comment]
Your web portfolio exudes professionalism and showcases your exceptional talent!

**Reply by reynoldputra** [reply]
Thank you! I hope we can work together on a project soon 🔥

**Comment by Syakhisk** [comment]
kewreen

**Reply by reynoldputra** [reply]
Inspired by https://al-az.me, Thanks bro

**Comment by dimasandhk** [comment]
keren banget mas

**Reply by reynoldputra** [reply]
Thanks :raised_hands:
